lib/index.ts

```diff
@@ -500,14 +500,24 @@ export default class Backburner {
         this._join(target, method, args);
       }
     } else {
-      let executeAt = this._platform.now() + wait || this._timers[index];
-      this._timers[index] = executeAt;
+      let executeAt = this._platform.now() + wait || _timers[index];
```

Comment:
I know this isn't new, but I don't fully understand how we ever expect `this._platform.now() + wait` to be falsey.

Reply:
yeah thats why I added another PR #359  to fix it, I can fix it here if you want